src/index.html

```diff
@@ -21,7 +21,360 @@
+
+      <article class="card">
+        <div class="card__image"></div>
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__code">Product code: 195434</p>
+
+        <section class="card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <div class="card__review">Reviews: 5</div>
+        </section>
+
+        <section class="card__price">
+          <div class="card__price card__price--header">Price:</div>
+          <div class="card__price card__price--amount">$2,199</div>
+        </section>
+
+        <a
+          href="#"
+          class="card__button"
+          data-qa="card-hover"
+        >
+          BUY
+        </a>
+      </article>
+
+      <article class="card">
+        <div class="card__image"></div>
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__code">Product code: 195434</p>
+
+        <section class="card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <div class="card__review">Reviews: 5</div>
+        </section>
+
+        <section class="card__price">
+          <div class="card__price card__price--header">Price:</div>
+          <div class="card__price card__price--amount">$2,199</div>
+        </section>
+
+        <a
+          href="#"
+          class="card__button"
+          data-qa="card-hover"
+        >
+          BUY
+        </a>
+      </article>
+
+      <article class="card">
+        <div class="card__image"></div>
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__code">Product code: 195434</p>
+
+        <section class="card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <div class="card__review">Reviews: 5</div>
+        </section>
+
+        <section class="card__price">
+          <div class="card__price card__price--header">Price:</div>
+          <div class="card__price card__price--amount">$2,199</div>
+        </section>
+
+        <a
+          href="#"
+          class="card__button"
+          data-qa="card-hover"
+        >
+          BUY
+        </a>
+      </article>
```

Comment:
Your cards contain a lot of duplicated code, which can make your HTML file harder to read and maintain. Consider using a templating engine or a JavaScript framework to generate these cards programmatically.